### ISSUE TITLE
check_patient_gtsp.R looks for sites and specimen info in same db group

### ISSUE THREAD

**Issue by cnobles** [BushmanLab/geneTherapyPatientReportMaker#55]
In our standard database schema, these two are kept in different locations and should point to two different places to get the information (specimen_management and intSites.db).

**Comment by anatolydryga** [comment]
yes, that was quick implementation, maybe adding 2 options: one for sites and one for specimen management with defaults

**Comment by cnobles** [comment]
Yeah, it was an easy fix to get it to work, but I wanted to make sure I didn't forget about it down the road. Good point though.
